CHANGELOG.md

```diff
@@ -25,6 +25,10 @@ Swift 5.6
 * [#3641]
 
   Adding a dependency requirement can now be done with the convenience initializer `.package(url: String, exact: Version)`.
+  
+* [#3890]
+
+ To increase the security of packages, SwiftPM performs trust on first use (TOFU) validation. The fingerprint of a package is now being recorded when the package is first downloaded from a Git repository or package registry. Subsequent downloads must have fingerpints matching previous recorded values, otherwise it would result in build warnings or failures depending on settings.   
```

Comment:
```suggestion
  To increase the security of packages, SwiftPM performs trust on first use (TOFU) validation. The fingerprint of a package is now being recorded when the package is first downloaded from a Git repository or package registry. Subsequent downloads must have fingerpints matching previous recorded values, otherwise it would result in build warnings or failures depending on settings.   
```